### ISSUE TITLE
Add WindowsSpyBlocker Filter into Adguard filters for Windows Tracking feature on Stealth Mode settings

### ISSUE THREAD

**Issue by validatedev** [AdguardTeam/AdguardFilters#62204]
### Prerequisites

Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**

- [*] I checked the documentation and found no answer
- [*] I checked to make sure that this issue has not already been filed

### Problem Description

As for improving the Windows tracking protection feature which was released on Adguard 7.5, we should bring this one step further, adding some Windows tracking and telemetry filters onto there should be a good move.

### Proposed Solution

Adding https://github.com/crazy-max/WindowsSpyBlocker into the filters. Right now, there isn't any compatible version for Adguard syntax of this filter on there: https://github.com/crazy-max/WindowsSpyBlocker/tree/master/data

**Comment by validatedev** [comment]
I tried using [simplewall](https://github.com/henrypp/simplewall) for that feature, but it interferes with Adguard so I uninstalled that.

**Comment by validatedev** [comment]
Anyone?

**Comment by Alex-302** [comment]
@validatedev Hi. Better ask here https://github.com/AdguardTeam/AdguardForWindows
Stealth Mode is not a filter.

**Comment by validatedev** [comment]
> @validatedev Hi. Better ask here https://github.com/AdguardTeam/AdguardForWindows
> Stealth Mode is not a filter.

Well, let me explain what is in my head:
The process of implementing such a feature that helps Stealth Mode should be this: The feature consists of two parts and that is just one part, if there isn’t any filter to enable and disable, then the second part isn’t implementable.
First, Adguard team should include (that sounds like I’m dictating lol, but you know that’s not the case) some filters that blocks the telemetry adresses-servers, and second part which is the selection option through in Stealth mode is other part, that isn’t involve the filter team.
In conclusion, I just suggest that there should be a filter that blocks Windows telemetry with referring the program which has been actively maintaining. Stealth mode thing is not your case, just want to refer that.
